src/passes/Memory64Lowering.cpp

```diff
@@ -46,20 +46,34 @@ struct Memory64Lowering : public WalkerPass<PostWalker<Memory64Lowering>> {
     ptr = builder.makeUnary(UnaryOp::WrapInt64, ptr);
   }
 
+  void ExtendAddress64(Expression*& ptr) {
```

Comment:
```suggestion
  void extendAddress64(Expression*& ptr) {
```
Sorry, I see I missed this in a past review in WrapAddress64, which should be `wrapAddress64` as well.

Reply:
renamed both